src/storage/files.cpp

```diff
@@ -55,10 +59,10 @@ files_t::read(const std::string& collection, const std::string& key) {
         );
     }
 
-    COCAINE_LOG_DEBUG(m_log, "reading object '%s'", key)(
-        "collection", collection,
-        "path", file_path
-    );
+    COCAINE_LOG_DEBUG(m_log, "reading object '{}'", key, attribute_list({
+        {"collection", collection}
+        // {"path", file_path}
```

Comment:
Think about how to implement old `operator<<` behavior for non-generic attributes.